libraries/CAN_comms/Src/CAN_comms.c

```diff
@@ -133,23 +141,25 @@ void CAN_comms_Tx_task(void* argument)
     /* Infinite loop */
     for(;;)
     {
+
         /* Wait until there is a message in the queue */ 
-        CAN_comms_Tx_msg_t CAN_comms_Tx_msg;
-        if (osOK == osMessageQueueGet(CAN_comms_Tx_queue, &CAN_comms_Tx_msg, NULL, osWaitForever))
-        {
+       CAN_comms_Tx_msg_t CAN_comms_Tx_msg;
+       if (osOK == osMessageQueueGet(CAN_comms_Tx_queue, &CAN_comms_Tx_msg, NULL, osWaitForever))
+       {
             /* Wait for a CAN mailbox semaphore to be released */
-            osSemaphoreAcquire(CAN_comms_Tx_mailbox_semaphore, osWaitForever);
+         osSemaphoreAcquire(CAN_comms_Tx_mailbox_semaphore, osWaitForever);
 
-            uint32_t can_mailbox; // Not used
-            if(HAL_OK != HAL_CAN_AddTxMessage(CAN_comms_config.hcan, &CAN_comms_Tx_msg.header, CAN_comms_Tx_msg.data, &can_mailbox))
-            {
-                return; // TODO: Error handle
-            }
-        }
-        else
+        uint32_t can_mailbox; // Not used
```

Comment:
Check it out in VSCode, still looks really unaligned. I had `git pull`ed
![image](https://github.com/user-attachments/assets/02a2ee8b-ab94-4978-9e51-f03398c2724c)